### ISSUE TITLE
Undefined offset: 1 - Could not authenticate

### ISSUE THREAD

**Issue by 0x1ad2** [pingpong-labs/twitter#16]
![image](https://cloud.githubusercontent.com/assets/1705904/6821625/b9c2e9da-d2d8-11e4-9b89-b0cf3b004944.png)

Some how I'll get an undefined offset of 1 when I try to authenticate. I've looked up the codebird class and printed some info.

```php
string(107) "<?xml version="1.0" encoding="UTF-8"?><errors><error code="32">Could not authenticate you.</error></errors>"
```

```php
array(23) {
  ["HTTP/1.1 401 Authorization Required"]=>
  string(0) ""
  ["cache-control"]=>
  string(62) "no-cache, no-store, must-revalidate, pre-check=0, post-check=0"
  ["content-length"]=>
  string(3) "107"
  ["content-type"]=>
  string(29) "application/xml;charset=utf-8"
  ["date"]=>
  string(29) "Wed, 25 Mar 2015 09:22:56 GMT"
  ["expires"]=>
  string(29) "Tue, 31 Mar 1981 05:00:00 GMT"
  ["last-modified"]=>
  string(29) "Wed, 25 Mar 2015 09:22:56 GMT"
  ["ml"]=>
  string(1) "A"
  ["pragma"]=>
  string(8) "no-cache"
  ["server"]=>
  string(5) "tsa_b"
  ["set-cookie"]=>
  string(100) "guest_id=v1%3A142727537648927922; Domain=.twitter.com; Path=/; Expires=Fri, 24-Mar-2017 09:22:56 UTC"
  ["status"]=>
  string(16) "401 Unauthorized"
  ["strict-transport-security"]=>
  string(17) "max-age=631138519"
  ["www-authenticate"]=>
  string(37) "OAuth realm="https://api.twitter.com""
  ["x-connection-hash"]=>
  string(32) "5f5286d62c1b552009574d8aca4c37fb"
  ["x-content-type-options"]=>
  string(7) "nosniff"
  ["x-frame-options"]=>
  string(10) "SAMEORIGIN"
  ["x-response-time"]=>
  string(1) "7"
  ["x-transaction"]=>
  string(16) "74d156d1485f2986"
  ["x-tsa-request-body-time"]=>
  string(1) "0"
  ["x-twitter-response-tags"]=>
  string(16) "BouncerCompliant"
  ["x-ua-compatible"]=>
  string(16) "IE=edge,chrome=1"
  ["x-xss-protection"]=>
  string(13) "1; mode=block"
}```

It looks like I could not authenticate...
I've provided the following config (and I published the config as you stated in the installation manual).

```php
return array(
	'consumer_key'		=>	'xAs5S******************* ',
	'consumer_secret'	=>  '572pd8As*********************************',

	'oauth_token'		=>	'73480072*********************************',
	'oauth_token_secret'=>  'uk6lsvF**************************************',

	'bearer_token'		=>	null,

	'callback_url'		=>  url('/twitter/login'),
    'fallback_url'      =>  url('/')
);```

Also don't I need to get some kind of modal for authentication?

**Comment by gravitano** [comment]
Sorry for my late response. I'll fix this. Thanks for telling me.

**Comment by 0x1ad2** [comment]
No problem @gravitano I'm using another package (https://github.com/thujohn/twitter) now because there wasn't a fix.

**Comment by KIVagant** [comment]
It already fixed or not yet?

**Comment by gravitano** [comment]
This issue have been solved in version `~2.0`. Thanks!
